rtree.go

```diff
@@ -146,10 +140,11 @@ func (tree *Rtree) bulkLoad(objs []Spatial) {
 	S := math.Floor(math.Sqrt(s))
 
 	// sort all entries by first dimension
-	sortByDim(int(h)%tree.Dim, entries)
-	tree.root = tree.omt(int(h), int(S), entries, int(s))
-	tree.size = n
+	sortByDim(0, entries)
```

Comment:
I'm not sure sorting by x-coordinate first is required. Two points that came to my mind:

1. The paper describes the algorithm for two dimensional data. The R-tree in this library is n-dimensional. 

2. The wording in the paper is `To do this, we need to sort the data items by turns x and y-coordinate in each other level. As we started to sort them by x-coordinate, in level h-2 we sort the data items by y-coordinate and in level h-4, we sort them by x-coordinate and so on.` I understand it as them trying to describe the general approach; to alternate the sorting dimension on every level. Sorting by x-coordinate seems to be an example rather than a requirement.